modules/lib/lib-schema/src/test/resources/test/UpdateDynamicContentSchemaHandlerTest.js

```diff
@@ -21,4 +21,403 @@ exports.updateInvalidContentSchemaType = function () {
+        "type": "CONTENT_TYPE",
+        "form": [
+            {
+                "formItemType": "OptionSet",
+                "name": "radioOptionSet",
+                "label": "Single selection",
+                "expanded": false,
+                "occurrences": {
+                    "maximum": 1,
+                    "minimum": 0
+                },
+                "selection": {
+                    "maximum": 1,
+                    "minimum": 1
+                },
+                "options": [
+                    {
+                        "name": "option_1",
+                        "label": "Option 1",
+                        "default": false,
+                        "items": [
+                            {
+                                "formItemType": "Input",
+                                "name": "text-input",
+                                "label": "Name",
+                                "helpText": "Text input",
+                                "maximize": true,
+                                "inputType": "TextLine",
+                                "occurrences": {
+                                    "maximum": 1,
+                                    "minimum": 1
+                                },
+                                "default": {
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9936826)
```suggestion
                                'default': {
```